cloud/amazon/ec2_customer_gateway.py

```diff
@@ -0,0 +1,239 @@
+#!/usr/bin/python
+#
+# This is a free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This Ansible library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this library.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: ec2_customer_gateway
+short_description: Manage an AWS customer gateway
+description:
+    - Manage an AWS customer gateway
+version_added: "2.1"
+author: Michael Baydoun (@MichaelBaydoun)
+requirements: [ botocore boto3 ]
+options:
+  bgp_asn:
+    description:
+      - Border Gateway Protocol (BGP) Autonomous System Number (ASN), required when state=present.
+    required: false
+    default: null
+  ip_address:
+    description:
+      - Internet-routable IP address for customers gateway, must be a static address.
+    required: true
+  name:
+    description:
+      - Name of the customer gateway.
+    required: true 
+  state:
+    description:
+      - Create or terminate the Customer Gateway.
+    required: false
+    default: present
+    choices: [ 'present', 'absent' ]
+extends_documentation_fragment:
+    - aws
+    - ec2
+'''
+
+EXAMPLES = '''
+
+# Note: You cannot create more than one customer gateway with the same IP address. If you run an identical request more than one time, the first request creates the customer gateway, and subsequent requests return information about the existing customer gateway. The subsequent requests do not create new customer gateway resources.
```

Comment:
Hi,

Applogies, I wasn't clear, you can specify a note in the following form
https://github.com/ansible/ansible-modules-extras/blob/devel/system/debconf.py#L32

So it appears as below the examples, as shown here http://docs.ansible.com/ansible/debconf_module.html